allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/NewChromeDriver.java

```diff
@@ -1,5 +1,6 @@
 package com.capgemini.ntc.selenium.core.newDrivers;
```

Comment:
please add in other drivers (Firefox, IE, etc.)
* elementHorizontalSlider

Reply:
They are there already.